example/README.md

```diff
@@ -73,7 +77,7 @@ rust: rustc 1.63.0-nightly
     fluence deploy
```

Comment:
Another thought. What if we add an additional step that is missing now and shows local testing with REPL using `fluence service repl echoService`? It's good to show a developer a full workflow.

The step could go either as a last step to the previous section or just before deployment.